dnfdaemon-server/session_manager.cpp

```diff
@@ -136,3 +140,13 @@ sdbus::MethodReply SessionManager::close_session(sdbus::MethodCall & call) {
 void SessionManager::start_event_loop() {
     connection->enterEventLoop();
 };
+
+void SessionManager::shut_down() {
```

Comment:
FWIW (not entirely sure what the flow is), it would definitely be good to log something like this here:
```C++
looger.info("Shutdown signal received, waiting for sessions to finish...");
```
As it may take some time? Might also want a timeout here, although we really don't want to interrupt an rpm transaction... Or is systemd taking care of the timeout?

Reply:
Log message definitely makes sense.
I'll take a look what could be done with systemd timeout. At least we can increase it's value - I think the default value is 90s.